libsrc/leddevice/schemas/schema-sk6812_ftdi.json

```diff
@@ -0,0 +1,59 @@
+{
+	"type": "object",
+	"required": true,
+	"properties": {
+		"output": {
+			"type": "string",
+			"title": "edt_dev_spec_outputPath_title",
+			"required": true,
+			"propertyOrder": 1
+		},
+		"rate": {
```

Comment:
Ist the baud rate totally dynamic or does it follow the USB serial baud rates?
If it follows the baud rate, I suggest you use a rateList as in
https://github.com/hyperion-project/hyperion.ng/blob/master/libsrc/leddevice/schemas/schema-adalight.json
to avoid user misconfiguration.

Reply:
it's not abut serial baud rate, since sk6812 is pwm driven and we simulate pwm signal using spi, allowing users to manually adjust baud rate will help to fix any troubles with dodgy strings. same for ws2812